### PR TITLE
Search page: Shows interviewer choices upon interview selection

### PR DESCRIPTION
**Pull request by apluscs**
1. search-1
2. search-2
3. search-3
4. search-4
5. search-5

### CODE REVIEW

File: src/main/webapp/search-interview-interviewers.jsp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <th scope="col">Job</th>
+     </tr>
+  </thead>
+  <tbody>
+     <tr>
+       <!-- TODO: get this info from a servlet -->
+        <td>Facebook</td>
+        <td>Program Manager</td>
+        <td>
+           <button type="button" class="btn btn-primary" data-company="Facebook" data-job="Program Manager" data-email="fbpm@gmail.com" onclick="selectInterview(this)">Select</button>
```
Comment (meagancheese) [inline src/main/webapp/search-interview-interviewers.jsp]: Long line

File: src/main/webapp/search-interview-interviewers.jsp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        </td>
+     </tr>
+     <tr>
+        <td>Google</td>
+        <td>Software Engineer</td>
+        <td> 
+           <button type="button" class="btn btn-primary" data-company="Google" data-job="Software Engineer" data-email="gswe@gmail.com" onclick="selectInterview(this)">Select</button>
```
Comment (meagancheese) [inline src/main/webapp/search-interview-interviewers.jsp]: Another long line

File: src/main/webapp/search-interview.html [hunk 3]
```diff
@@ -118,13 +118,32 @@ <h5 class="text-center lead">Select the time you want from the dropdown on the d
...
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="modal-title">Modal title</h5>
+              <button type="button" class="close" data-dismiss="modal">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body" id="modal-body">
+
```
Comment (meagancheese) [inline src/main/webapp/search-interview.html]: Unnecessary line

File: src/main/webapp/search-interview.js [hunk 4]
```diff
@@ -24,11 +24,18 @@ function loadInterviews() {
 }
 
 // Confirms interview selection with user and sends this selection to Datastore if confirmed.
-function selectInterview() {
-  // TODO: fill in with actual interview info.
-  if (confirm("You selected: Sunday 7/5 from 6:30 PM - 7:30 PM. Click OK if you wish to proceed.")) {
-    alert("You have scheduled an interview on Sunday 7/5 from 6:30 PM - 7:30 PM. Check your email for more information.");
-    // TODO: Call a servlet to handle this selection.
+function selectInterview(interviewer) {
+  if (confirm(`You selected: Sunday 7/5 from 6:30 PM - 7:30 PM with a ${interviewer.getAttribute("data-company")} ${interviewer.getAttribute("data-job")}. Click OK if you wish to proceed.`)) {
```
Comment (meagancheese) [inline src/main/webapp/search-interview.js]: There's a way to continue text over several js lines. I think it's a type of slash at the end of the line.

Comment (meagancheese) [inline src/main/webapp/search-interview.js]: Change from ticks to '.

File: src/main/webapp/search-interview.js [hunk 5]
```diff
@@ -24,11 +24,18 @@ function loadInterviews() {
...
+    alert(`You have scheduled an interview on Sunday 7/5 from 6:30 PM - 7:30 PM with a ${interviewer.getAttribute("data-company")} ${interviewer.getAttribute("data-job")}. Check your email for more information.`);
```
Comment (meagancheese) [inline src/main/webapp/search-interview.js]: Same comments as above.

File: src/main/webapp/search-interview.js (hunk 4 above)
Comment (meagancheese) [inline src/main/webapp/search-interview.js]: See if it's possible to replace " with ' for getAttribute calls.

**Review by meagancheese — CHANGES_REQUESTED** [review]
Some comments, also add a picture in the description.

File: src/main/webapp/search-interview.js (hunk 4 above)
Comment (apluscs) [reply src/main/webapp/search-interview.js]: Need the backticks for the string interpolation.

Comment (apluscs) [reply src/main/webapp/search-interview.js]: Gonna string concatenate based off of https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

File: src/main/webapp/search-interview-interviewers.jsp [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<table class="table">
+  <thead>
+     <tr>
+        <th scope="col">Company</th>
+        <th scope="col">Job</th>
```
Comment (meagancheese) [inline src/main/webapp/search-interview-interviewers.jsp:5]: Nit: center these?

**Review by meagancheese — CHANGES_REQUESTED** [review]
Small tweak, otherwise LGTM.

**Review by meagancheese — CHANGES_REQUESTED** [review]
Change "Modal Title" to something meaningful.

**Review by meagancheese — APPROVED** [review]
LGTM

File: src/main/webapp/search-interview.html [hunk 7]
```diff
@@ -106,25 +105,44 @@ <h5 class="text-center lead">Select the time you want from the dropdown on the d
...
-              <!-- TODO: change onclick to addEventListener inside loadInterviews() so it can pass in params. -->
-              <button type="button" class="btn btn-primary mb-2" onclick="selectInterview()">Select</button>
+              <button type="button" class="btn btn-primary mb-2" 
+              onclick="showInterviewers(this)" id="7/5/2020">Select</button>
             </div>
           </div>
         </form>
         <br>  
       </section>
+
```
Comment (GrantComm) [inline src/main/webapp/search-interview.html]: Is this space necessary?

**Review by GrantComm — APPROVED** [review]
LGTM

File: src/main/webapp/search-interview.html (hunk 7 above)
Comment (apluscs) [reply src/main/webapp/search-interview.html]: l guess not but l like how it separates the modal from the other stuff.

File: src/main/webapp/search-interview.html [hunk 8]
```diff
@@ -106,25 +105,43 @@ <h5 class="text-center lead">Select the time you want from the dropdown on the d
...
+              <select class="form-control" id="7/5/2020-options">
                 <option>9:30 AM - 10:30 AM</option>
                 <option>6:30 PM - 7:30 PM</option>
               </select>
             </div>
             <div class="col-4">
-              <!-- TODO: change onclick to addEventListener inside loadInterviews() so it can pass in params. -->
-              <button type="button" class="btn btn-primary mb-2" onclick="selectInterview()">Select</button>
+              <button type="button" class="btn btn-primary mb-2" 
+              onclick="showInterviewers(this)" id="7/5/2020">Select</button>
```
Comment (jalexanderqed) [inline src/main/webapp/search-interview.html:122]: I think this line (starting with onclick) should be indented one more space.

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**